content/topics/clean-code/general/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: General Clean Code Guidelines
+ready: True
+---
+
+"It's easy to write code that computers understand, but hard to write code that other people understand ". -Some Smart Person
```

Comment:
“Any fool can write code that a computer can understand. Good programmers write code that humans can understand.”
― Martin Fowler